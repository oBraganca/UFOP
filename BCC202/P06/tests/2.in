1 24
1 38
1 79
1 6
1 75
6
2 785
6
2 48
2 87
2 3
3
4 99
2 68
4 27
1 47
5
4 13
1 97
6
5
5
-1