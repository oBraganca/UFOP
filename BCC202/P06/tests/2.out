Posicao inicial 0
Posicao inicial 0
Nao Pertence
Nao Pertence
6 6 47 75 79 
Nao Pertence
Posicao inicial 1
6 6 47 75 79 97 
6 6 47 75 79 97 
